lib/gwdb-legacy/database.ml

```diff
@@ -892,53 +939,96 @@ let opendb bname =
     make_record_exists (snd patches.h_family) (snd pending.h_family)
       families_len
   in
+  let ( im_persons,
+        im_ascends,
+        im_unions,
+        im_families,
+        im_couples,
+        im_descends,
+        im_strings )
+        : ro_data_records =
+    let bfname = Unix.realpath bname in
```

Comment:
@Keryan-dev : today I encounter build problems on Debian 10/Ocaml 4.09.1 with this `Unix.realpath` function that is in the Ocaml API since version 4.13 only. Is there an alternative we can use?

Reply:
To be a bit more comprehensive : the name (path) of the base that was formerly used is not robust enough for proper identification. For example `./base.gwd` would not be the same as `base.gwd` or even `base`. This was somehow fine until this PR since the server had one way to refer to paths, but now that a user may enter one through the new CLI option.

Also, restricting to `Filename.basename` would be simpler but could also lead to issues if the server has access to several bases with the same name in different directories.